Utilities/CommonDriver.cs

```diff
@@ -0,0 +1,14 @@
+﻿using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MarsLanguageSkillsSpecflow.Utilities
+{
+    public class CommonDriver
```

Comment:
You will need to create a initialise method and then use Hooks to call that method 

Public static IWebDriver driver;

Public void initailise(){
            driver = new ChromeDriver();

}


And your hooks class will call that initailise() method

Please see Hooks you will need to add hooks!
https://docs.specflow.org/projects/specflow/en/latest/Bindings/Hooks.html

https://codebun.com/how-to-implement-hooks-in-specflow-c/